.schema/embed.go

```diff
@@ -0,0 +1,9 @@
+package schema
+
+import "embed"
+
+//go:embed *.json all:pipeline
```

Comment:
What does `all:pipeline` do? :D

Reply:
Adds all files from the pipeline directors.